File: app/src/main/java/ir/touristland/Models/HotelPrices/JsonMember970521.java
package ir.touristland.Models.HotelPrices;

import com.google.gson.annotations.SerializedName;

public class JsonMember970521 {

    @SerializedName("date")
    private Date date;

    @SerializedName("boardPrice")
    private int boardPrice;

    @SerializedName("webservicePrice")
    private int webservicePrice;

    @SerializedName("extraPersonPrice")
    private int extraPersonPrice;

    @SerializedName("salesPrice")
    private int salesPrice;

    @SerializedName("reserveStatus")
    private boolean reserveStatus;

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public int getBoardPrice() {
        return boardPrice;
    }

    public void setBoardPrice(int boardPrice) {
        this.boardPrice = boardPrice;
    }

    public int getWebservicePrice() {
        return webservicePrice;
    }

    public void setWebservicePrice(int webservicePrice) {
        this.webservicePrice = webservicePrice;
    }

    public int getExtraPersonPrice() {
        return extraPersonPrice;
    }

    public void setExtraPersonPrice(int extraPersonPrice) {
        this.extraPersonPrice = extraPersonPrice;
    }

    public int getSalesPrice() {
        return salesPrice;
    }

    public void setSalesPrice(int salesPrice) {
        this.salesPrice = salesPrice;
    }

    public boolean isReserveStatus() {
        return reserveStatus;
    }

    public void setReserveStatus(boolean reserveStatus) {
        this.reserveStatus = reserveStatus;
    }

    @Override
    public String toString() {
        return
                "JsonMember970521{" +
                        "date = '" + date + '\'' +
                        ",boardPrice = '" + boardPrice + '\'' +
                        ",webservicePrice = '" + webservicePrice + '\'' +
                        ",extraPersonPrice = '" + extraPersonPrice + '\'' +
                        ",salesPrice = '" + salesPrice + '\'' +
                        ",reserveStatus = '" + reserveStatus + '\'' +
                        "}";
    }
}

File: app/src/main/java/ir/touristland/Models/HotelPrices/JsonMember970521.kt
package ir.touristland.Models.HotelPrices

import com.google.gson.annotations.SerializedName

class JsonMember970521 {

    @SerializedName("date")
    var date: Date? = null

    @SerializedName("boardPrice")
    var boardPrice: Int = 0

    @SerializedName("webservicePrice")
    var webservicePrice: Int = 0

    @SerializedName("extraPersonPrice")
    var extraPersonPrice: Int = 0

    @SerializedName("salesPrice")
    var salesPrice: Int = 0

    @SerializedName("reserveStatus")
    var isReserveStatus: Boolean = false

    override fun toString(): String {
        return "JsonMember970521{" +
                "date = '" + date + '\''.toString() +
                ",boardPrice = '" + boardPrice + '\''.toString() +
                ",webservicePrice = '" + webservicePrice + '\''.toString() +
                ",extraPersonPrice = '" + extraPersonPrice + '\''.toString() +
                ",salesPrice = '" + salesPrice + '\''.toString() +
                ",reserveStatus = '" + isReserveStatus + '\''.toString() +
                "}"
    }
}